### PR TITLE
Add disclaimer to http2 introduction

### PR DESCRIPTION
**Pull request by surma**
As per @igrigorik’s [comment](https://github.com/google/WebFundamentals/pull/3547#issuecomment-250797047)

### CODE REVIEW

**Comment by igrigorik** [comment]
@surma can we also please link the title to the O'Reilly page for the book? Link: http://shop.oreilly.com/product/0636920028048.do

**Comment by igrigorik** [comment]
Thanks @surma. LGTM.

File: src/content/en/fundamentals/performance/http2/index.md [hunk 1]
```diff
@@ -10,6 +10,11 @@ description: HTTP/2 (or h2) is a binary protocol that brings push, multiplexing
 {% include "web/_shared/contributors/ilyagrigorik.html" %}
 {% include "web/_shared/contributors/surma.html" %}
 
+Note: The following content is an excerpt from [High Performance Browser
+Networking](http://shop.oreilly.com/product/0636920028048.do) (O'Reilly, Ilya
+Grigorik). For full version and related content, see
+[hpbn.com](https://hpbn.co/http2/).
```
Comment (igrigorik) [inline src/content/en/fundamentals/performance/http2/index.md]: Err, "hpbn.co" not com. Let's link to the top level page ".co/".

**Comment by surma** [comment]
My bad, fixed

**Pull request merged**